src/libraries/Microsoft.Extensions.HostFactoryResolver/pkg/Microsoft.Extensions.HostFactoryResolver.Sources.pkgproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <PropertyGroup>
+    <IsShipping>false</IsShipping>
```

Comment:
Just wondering do we plan ever shipping this in NuGet, or will we just use it internally to share between other dotnet repos using our feeds?

Reply:
Never on nuget.  Just for sharing with ASP.NET and EFCore.